plugins/node/opentelemetry-instrumentation-express/src/types.ts

```diff
@@ -88,6 +98,7 @@ export interface ExpressInstrumentationConfig extends InstrumentationConfig {
   ignoreLayers?: IgnoreMatcher[];
   /** Ignore specific layers based on their type */
   ignoreLayersType?: ExpressLayerType[];
+  spanNameHook?: SpanNameHook;
```

Comment:
We will need to update the doc in the readme too

Reply:
Updated README